yellow-paper/docs/cryptography/protocol-overview.md

```diff
@@ -0,0 +1,113 @@
+
+#### Protogalaxy Fold
+
+The "Fold" Prover/Verifier validates that `k` instances of a defined relation (in our case the Honk relation) have been correctly folded into an accumulator instance.
+
+#### Protogalaxy Decider
+
+The "Decider" Prover/Verifier valiate whether an accumulator instance correctly satisfies the accumulator relation.
+
+## Goblin Plonk
+
+[Goblin Plonk](https://hackmd.io/@aztec-network/BkGNaHUJn/%2FGfNR5SE5ShyXXmLxNCsg3g) is a computation delegation scheme that improves Prover performance when evaluating complex algorithms.
+
+In the context of an IVC scheme, Goblin Plonk enables a Prover to defer non-native group operations required by a Verifier algorithm, across multiple recursive proofs, to a single step evaluated at the conclusion of the IVC Prover algorithm.
+
+Goblin Plonk is composed of three subcomponents:
+
+#### Transcript Aggregation Subprotocol
+
+This subprotocol aggregates deferred computations from two independent instances, into a single instance
+
+#### Elliptic Curve Virtual Machine (ECCVM) Subprotocol
+
+The ECCVM is a Honk circuit with a custom circuit arithmetisation, designed to optimally evaluate elliptic curve arithmetic computations that have been deferred. It is defined over the Grumpkin elliptic curve
+
+#### Translator Subprotocol
+
+The Translator is a Honk circuit with a custom circuit arithmetisation, designed to validate the input commitments of an ECCVM circuit align with the delegated computations described by a Goblin Plonk transcript commitment
+
+## Plonk Data Bus
+
+The [Plonk Data Bus](https://aztecprotocol.slack.com/files/U8Q1VAX6Y/F05G2B971FY/plonk_bus.pdf) protocol enables efficient data transfer between two Honk instances within a larger IVC protocol.
+
+# Polynomial Commitment Schemes
+
+The UltraPlonk, Honk, Goblin Plonk and Plonk Data Bus protocols utilize Polynomial Interactive Oracle Proofs as a core component, neccessitating the use of polynomial commitment schemes (PCX).
+
+UltraPlonk and Honk utilize multilinear PCS. The Plonk Data Bus and Goblin Plonk also utilize univariate PCS.
+
+For multilinear polynomial commitment schemes, we use the [ZeroMorph](https://eprint.iacr.org/2023/917) protocol, which itself uses a univariate PCS as a core component.
+
+Depending on context we use the following two univariate schemes within our cryptography stack
+
+## KZG Commitments
+
+The [KZG](https://www.iacr.org/archive/asiacrypt2010/6477178/6477178.pdf) polynomial commitment scheme requires a universal setup and is instantiated over a pairing-friendly elliptic curve.
+
+Computing an opening proof of a degree-$n$ polynomial requires $n$ scalar multiplications, with a constant proof size and a constant verifier time.
+
+## Inner Product Argument
+
+The [IPA](https://eprint.iacr.org/2019/1177.pdf) PCS has worse asymptotics than KZG but can be instantiated over non-pairing friendly curves.
```

Comment:
state this is needed for Grumpkin, if we are at it maybe we should also say we use cycle of curves somewhere?